setup.py

```diff
@@ -1,27 +1,64 @@
+# pylint: disable=C0114
+import re
+
+from pkg_resources import get_distribution, DistributionNotFound
 from setuptools import setup, find_packages
 
 long_description = """A library for image augmentation in machine learning experiments, particularly convolutional
 neural networks. Supports the augmentation of images, keypoints/landmarks, bounding boxes, heatmaps and segmentation
 maps in a variety of different ways."""
 
+INSTALL_REQUIRES = [
+    "six",
+    "numpy>=1.15",
+    "scipy",
+    "Pillow",
+    "matplotlib",
+    "scikit-image>=0.14.2",
+    "opencv-python-headless",
+    "imageio",
+    "Shapely",
+]
+
+ALT_INSTALL_REQUIRES = {"opencv-python-headless": "opencv-python"}
+
+
+def check_alternative_installation(install_require, alternative_install_require):
+    """If some version version of alternative requirement installed, return alternative,
+    else return main.
+    """
+    try:
+        alternative_pkg_name = re.split(r"[!<>=]", alternative_install_require)[0]
+        get_distribution(alternative_pkg_name)
+    except DistributionNotFound:
+        return install_require
```

Comment:
This should then expect `alternative_install_require` to be a list and iterate over it via a for loop.
If an alternative name in a loop iteration is found, return that name. Otherwise return `install_require` at the end.